### ISSUE TITLE
Build errors for version 8.3.8

### ISSUE THREAD

**Issue by mg** [Pushwoosh/pushwoosh-phonegap-plugin#380]
Hi. I am having build problems and after having removed and added back every plugin in my project, I've concluded that it is the pushwoosh plugin that is causing the problem:
![image](https://user-images.githubusercontent.com/1972/196748002-140c8cf2-8c57-435c-8ea7-028cc60c4c1d.png)
Running with the suggested options (stacktrace,scan,etc) doesnt provide any further information. I am using cordova 11 and cordova-android 11.0.0.

Is this something you are familiar with? This is the config 
![image](https://user-images.githubusercontent.com/1972/196748295-8f977f19-ef2d-480d-9885-e3a8853ee836.png)

**Comment by mg** [comment]
Update: this only happens when building for Android. Building for iOS completes without errors.

**Comment by wfhm** [comment]
@mg Apologies for such a delayed reply! I just tried reproducing the issue with the latest (8.3.8) version of the plugin and it worked just fine with Cordova 11 and cordova-android@11.0.0. If the issue persists, could you please share a simple reproducer project with us? You can do it by creating a [support ticket](https://help.pushwoosh.com/hc/en-us/requests/new) and mentioning this issue in the description.

**Comment by mg** [comment]
Hi @wfhm and thanks for the response!

I've sort of figured out what was happening. Your plugin and the cordova-plugin-firebase-analytics plugin have some shared dependencies. When I added either one to my project building succeeded, but when I added both of them together it failed because the project was now trying to fetch some dependency from jumio.com and Jumio rejected the request because I was running on the corporate network behind a product called WebSense that does funny things to our SSL certificates.
